### PR TITLE
Improved: Button to export OMPL of a category

### PR DESCRIPTION
**Pull request by math-GH**
Ref #6517
Ref #6312

grafik

How to test the feature manually:

1. go to category
2. export OPML file

Pull request checklist:

- [x] clear commit messages
- [x] code manually tested

### CODE REVIEW

**Comment by math-GH** [comment]
Milestone `1.24.1` or waiting for `1.25.0`

**Comment by Alkarex** [comment]
I do not see any risk of regression here, so fine for 1.24.1, and thanks 👍🏻

**Comment by math-GH** [comment]
It is something between an improvement and a fix that I forget to add this button in category too ;)

File: app/views/helpers/category/update.phtml [hunk 1]
```diff
@@ -243,5 +245,18 @@
 				</div>
 			</div>
 		</fieldset>
+
+		<fieldset>
+			<legend><?= _t('sub.import_export.export') ?></legend>
+			<div class="form-group">
+				<label class="group-name"><?= _t('sub.feed.export-as-opml.label') ?></label>
+				<div class="group-controls">
+					<a class="btn btn-important" href="<?= _url('index', 'opml', 'get', 'c_'.$this->category->id()) ?>" download="<?= 'f_'.$this->category->id()?> <?= $this->category->name() ?>.opml.xml">
```
Comment (Alkarex) [inline app/views/helpers/category/update.phtml]:
```suggestion
					<a class="btn btn-important" href="<?= _url('index', 'opml', 'get', 'c_'.$this->category->id()) ?>" download="f_<?= $this->category->id() ?> <?= $this->category->name() ?>.opml.xml">
```

Minor consistency

**Review by Alkarex — APPROVED** [review]

File: app/views/helpers/category/update.phtml (hunk 1 above)
Comment (math-GH) [reply app/views/helpers/category/update.phtml]: one more thing. it is `c_` instead of `f_`

**Pull request merged**